### DISCUSSION TITLE
Hello World project with ESP Wrover Kit 4.1

### DISCUSSION THREAD

**Discussion by soky** [Mair/esp32-course#120] [Course Qeustions]
When building the Hello World project for the ESP WROVER Kit 4.1 it returns "Hello World" but when Flash it, Hello World does not show on the ESP WROVER KIT 4.1 just a blank screen.  When Choosing boards, under the entry for ESP WROVER Kit 4.1 it has two options 3.8 and 1.3 (I think thats correct). Which one should I choose.

soky157

**Comment by Mair** [comment]
Hi Soky,
The only thing I can think of is you might need to configure the jumpers to enable UART see https://docs.espressif.com/projects/esp-idf/en/latest/esp32/hw-reference/esp32/get-started-wrover-kit.html#setup-options

**Reply by soky** [reply]
Thank you Mair, I will try that.
